engine/src/flutter/impeller/display_list/aiks_dl_unittests.cc

```diff
@@ -1119,5 +1121,26 @@ TEST_P(AiksTest, DisplayListToTextureAllocationFailure) {
   EXPECT_EQ(texture, nullptr);
 }
 
+TEST_P(AiksTest, DisplayListToTextureWithMipGenerationOnGLES) {
+  if (GetBackend() != PlaygroundBackend::kOpenGLES) {
+    GTEST_SKIP() << "Only relevant for GLES";
+  }
```

Comment:
This wouldn't work for other backends though?

Reply:
Right, other backends won't fail this case.